test/functional/tools/for_workflows/create_input_collection.xml

```diff
@@ -0,0 +1,42 @@
+<tool id="create_input_collection" name="create_input_collection" version="0.1.0">
+  <description>This tool is used to create a collection of text files.</description>
+  <command detect_errors="exit_code">
+    mkdir outputs; cd outputs; python $script
+  </command>
+  <configfiles>
+    <configfile name="script">
+import os
+
+for i in range($collection_size):
+    template = "File number %s\n"
+    contents = template % i
+    with open(str(i), "w") as f:
+        f.write(contents)
+</configfile>
```

Comment:
It does strip commands - but it doesn't strip configfiles in tool parsing or evaluation as far as I can see (quickly reviewing the path through the code). I'd see stripping configfiles as bad behavior - it is described as a file so the content should be describing the full file contents.